mava/utils/environments/flatland_utils.py

```diff
@@ -22,7 +22,27 @@
 )
 from mava.wrappers.flatland import FlatlandEnvWrapper
 
-try:
+
+def check_flatland_import() -> bool:
```

Comment:
Is this something we only need for Flatland?

Reply:
So we can create these functions for other imports, flatland is just the one that causes the most issues (mainly because it is so strict on package versions).

Reply:
Ok I see, thanks 👍